packages/server/src/mdx.tsx

```diff
@@ -7,16 +7,16 @@ import matter from 'gray-matter';
 import type { GetStaticPathsContext, GetStaticPropsContext, GetStaticPropsResult } from 'next';
 import { appWithTranslation } from 'next-i18next';
 import { serverSideTranslations } from 'next-i18next/serverSideTranslations.js';
-import { dirname, join, resolve } from 'path';
-import * as React from 'react';
+import { dirname, resolve } from 'path';
+import React, { Fragment } from 'react';
 import { getHighlighter } from 'shiki';
 import { IS_PRODUCTION } from './constants';
 import { getSlug } from './routes';
 import { SerializeTOC } from './toc';
 
-const Provideri18n = appWithTranslation(({ children }) => <React.Fragment children={children} />);
+const Provideri18n = appWithTranslation(() => <Fragment />);
```

Comment:
don't know if it's right, but there is no type issue no type issue

Reply:
where `<Provideri18n>` gets used?